nservicebus/testing/index.md

```diff
@@ -39,7 +39,7 @@ snippet:TestingSaga
 
 ## Configuring unobtrusive message conventions
 
-If you are using [unobtrusive mode](/nservicebus/messaging/unobtrusive-mode.md) you need to configure the unit test support with those conventions as shown below.
+Prior to Version 6, if you are using [unobtrusive mode](/nservicebus/messaging/unobtrusive-mode.md) you need to configure the unit test support with those conventions as shown below.
```

Comment:
See the proper version specific language, see https://github.com/particular/docs.particular.net#version-language
It should be something like 
For Versions 5 and below.

Also remove the usage of you.